mod-record-specifications-server/src/main/java/org/folio/rspec/controller/SpecificationStorageController.java

```diff
@@ -56,6 +57,12 @@ public ResponseEntity<SpecificationDtoCollection> getSpecifications(Family famil
     return ResponseEntity.ok(specifications);
   }
 
+  @Override
+  public ResponseEntity<SpecificationDto> getSpecification(UUID specificationId, IncludeParam include) {
+    var specification = specificationService.findSpecificationById(specificationId, include);
```

Comment:
Minor.
For me more natural method name in @Service is getSpecificationById(...) instead of "find..." (looks good for @Repository)